data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,277 @@ mission "Arachi Cartoonist 2"
+			`	It's a very tight race, but the Saryds begin slowing once they approach the wall, both out of exhaustion and to prevent crashing into the wall. As the Saryds arrive at the base of the cliff, they immediately lay down to rest while the Arach on their team begins to unstrap the chariot, and the Kimek hurriedly begins climbing the wall with the bags they had packed earlier.`
+			`	Before long, the Kimek have all reached the top of the cliff and begin opening their bags to grab a multitude of paddles, some kind of pole which looks vaguely like a combination shovel and a pickaxe, and a long rope. "To cross the river, the primary goal of the game is," Pairri explains. "Prepare the raft for their team, the Kimek must. My role this would have been. As skilled in anticipating the actions of my friends I hope my former opponent is."`
+			`	Back at the base of the cliff, the Arachi are done helping their Saryd teammates out of the chariot straps, and together they prepare for the climb themselves. Shockingly, the Saryds lay across the smaller Arachi's backs, and together they use the lengths to fabric to secure themselves to each other. It seems the Arachi carry their larger teammates! You have your monitor focus on Pairri's friends again, and see that Ulmodi is just finishing checking his safety straps, after which Tumub begins his climb. Despite the stress of carrying someone much larger than him, Tumub's movements remain precise and deliberate, never more than one leg leaving the wall at a time as he carefully chooses the best spots to anchor himself and support the weight of his Saryd friend.`
+			`	Once Tumub reaches the top, Ulmodi gets dislodges himself and removes the harness from Tumub, before then lifting his Arach teammate into his arms so that he may climb onto the Saryd's back this time, as Ulmodie races to the by now nearly complete raft. Tumub climbs from Ulmodi's back into the raft and both their Kimek teammate finish properly securing the ropes. Once done, Ulmodi and the Kimek begin pushing the raft to the river, where Ulmodi now lifts up the Kimek and leaps into the raft just after it begins to float on the water.`
+			`	The Kimek quickly picks up three of the paddles and is working to fight the river's flow, so far without much success. Meanwhile Ulmodi grabs the pole tool from between two of the logs and uses the pick edges to grab onto some of rocks jutting from the river, in order to stabilize the raft and help pull them across the swift current, while alternately thrusting at the rocks with the shovel portion in order to keep their small raft from dashing against them. About halfway across, Tumub has caught his breath and grabs the four remaining paddles in order to help the Kimek. As they both struggle with them against the current, the paddles begin to actually make a difference, and the raft both becomes more stable and begins to make it's way across the river even when Ulmodi has no convenient rock to pull against.`
```

Comment:
```suggestion
			`	The Kimek quickly picks up three of the paddles and is working to fight the river's flow, so far without much success. At the same time, Ulmodi grabs the pole tool from between two of the logs and uses the pick edges to grab onto some of rocks jutting from the river, while alternately thrusting at other rocks with the shovel portion. This has the effect of stabilizing the raft, as well as helping to pull them across the swift current and keeping their small raft from dashing against the rocks. About halfway across, Tumub has caught his breath and grabs the four remaining paddles in order to help the Kimek. As they both struggle with them against the current, the paddles begin to actually make a difference, and the raft begins to make its way across the river even when Ulmodi has no convenient rock to pull against.`
```

Reply:
Since you suggested two different changes to this line, should I pick whichever? Is one preferrable to the other?

Reply:
Oh, my bad, that was unintentional. I'll delete one of them then.